docs/guides/tfhers/serialization.md

```diff
@@ -6,56 +6,51 @@ Concrete already has its serilization functions (e.g. `tfhers_bridge.export_valu
 
 ## Ciphertexts
 
-We can deserialize `FheUint8` (and similarly other types) using `bincode`
+We should deserialize `FheUint8` using safe serialization functions from TFHE-rs
 
 ```rust
 use tfhe::FheUint8;
+use tfhe::safe_serialization::{safe_deserialize, safe_serialize};
+
+const SERIALIZE_SIZE_LIMIT: u64 = 1_000_000_000;
 
 /// ...
 
 fn load_fheuint8(path: &String) -> FheUint8 {
-    let path_fheuint: &Path = Path::new(path);
-    let serialized_fheuint = fs::read(path_fheuint).unwrap();
-    let mut serialized_data = Cursor::new(serialized_fheuint);
-    bincode::deserialize_from(&mut serialized_data).unwrap()
+    let file = fs::File::open(path).unwrap();
+    safe_deserialize(file, SERIALIZE_SIZE_LIMIT).unwrap()
 }
 ```
 
 To serialize
 
 ```rust
 fn save_fheuint8(fheuint: FheUint8, path: &String) {
-    let mut serialized_ct = Vec::new();
-    bincode::serialize_into(&mut serialized_ct, &fheuint).unwrap();
-    let path_ct: &Path = Path::new(path);
-    fs::write(path_ct, serialized_ct).unwrap();
+    let file = fs::File::create(path).unwrap();
+    safe_serialize(fheuint, file, SERIALIZE_SIZE_LIMIT).unwrap()
 }
 ```
 
 ## Secret Key
 
-We can deserialize `LweSecretKey` using `bincode`
+TFHE-rs safe serialization doesn't yet support this key so we should deserialize `LweSecretKey` using `bincode`
```

Comment:
cc @IceTDrinker we really need this before considering this feature as stable

Reply:
this has been done I think ?

Reply:
Pinged Nico